scripts/ci/validate-typescript-api.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
Even tho it's almost sure that we'll use this file or a similar one soon, there is no use for it right now (also I spot part of it that need to be updated either way), so let's remove this too.

Reply:
Removed this as well.